e2e/playwright/test-utils.ts

```diff
@@ -585,6 +585,15 @@ export async function getUtils(page: Page, test_?: typeof test) {
       })
     },
 
+    /**
+     * @deprecated Sorry I don't have time to fix this right now, but runs like
+     * the one linked below show me that setting the open panes in this manner is not reliable.
+     * You can either set `openPanes` as a part of the same initScript we run in setupElectron/setup,
+     * or you can imperatively open the panes with functions like {openKclCodePanel}
+     * (or we can make a general openPane function that takes a paneId).,
+     * but having a separate initScript does not seem to work reliably.
+     * @link https://github.com/KittyCAD/modeling-app/actions/runs/10731890169/job/29762700806?pr=3807#step:20:19553
+     */
```

Comment:
Ah, dang :(

Reply:
Ah yeah I'm sorry I still believe in the spirit of what you're going for so I didn't want to rip it out! I just think it needs to have different implementation details.

Reply:
And I might be off and it's something about the `page.reload()` or something else in that function, I just saw it was a common thread between the tests failing across the PRs.

Reply:
Yeah yeah all good :heart: